install/templates/step1.php

```diff
@@ -0,0 +1,35 @@
+<?php include __DIR__ . '/header.php'; ?>
+
+    <h2><?= _('Welcome to the installation of Poweradmin!') ?></h2>
```

Comment:
http://php.net/manual/en/language.basic-syntax.phptags.php
Since PHP 5.4 this tag is always available regardless of the short_open_tag ini setting.